scripts/package/package_android_ndk.sh

```diff
@@ -63,7 +62,7 @@ if [ ! -e "${TARGET_PATH}/${ANDROID_NDK_BASENAME}.tar.gz" ]; then
 	# Cleanup the package, shrinking 2.9GB down to 1.6GB (extracted)
 	echo "Cleaning NDK" ${TMP}/${ANDROID_NDK}
 
-	(cd ${TMP} && rm -rf ${ANDROID_NDK}/prebuilt)
+	# keep: (cd ${TMP} && rm -rf ${ANDROID_NDK}/prebuilt) -- it's easier to rebuild packages using NDKs cmake
```

Comment:
it is a pretty small folder (~7mb), but having it helps to rebuild packages easier

Reply:
But for those things, we could as easily require the developer to have the NDK installed, and ANDROID_HOME etc setup?

We'll support that soon anyways so that they don't have to create/install this package. So I think that it's still something to think about, since the Docker disc space is still an issue.

Reply:
I can do removing of that folder on extender side in Dockerfile then in [my PR ](https://github.com/defold/extender/pull/253/files)